src/frontend/src/pages/HomePage/Home.tsx

```diff
@@ -24,7 +25,10 @@ const Home = () => {
 
   return (
     <PageLayout title="Home" hidePageTitle>
-      <Typography variant="h3" sx={{ my: 2, textAlign: 'center', pt: 3 }}>
+      <Box marginTop={2}>
+        <NavTopBar />
+      </Box>
+      <Typography variant="h3" sx={{ marginTop: 0, textAlign: 'center', pt: 3, padding: 0 }}>
```

Comment:
see how the welcome message looks moved up next to the name/ dropdown, I think it looks aight like this, but I wanna play around with it a bit while we're moving stuff